apps/torus-allocator/src/app/layout.tsx

```diff
@@ -29,9 +29,14 @@ export default function RootLayout({
 }: Readonly<{
   children: React.ReactNode;
 }>) {
+  const isMainnet = env("NEXT_PUBLIC_TORUS_CHAIN_ENV") === "mainnet";
+  const domains = isMainnet
+    ? "allocator.torus.network,rollup.torus.network"
+    : "allocator.testnet.torus.network,testnet.rollup.torus.network";
```

Comment:
This should be parameterized: we should be able to use other potential environments like `"dev"`.

```suggestion
  const chainEnv = env("NEXT_PUBLIC_TORUS_CHAIN_ENV")
  const isMainnet = chainEnv === "mainnet";
  const domains = isMainnet
    ? "allocator.torus.network,rollup.torus.network"
    : "allocator.${chainEnv}.torus.network,testnet.${chainEnv}.torus.network";
```

Actually, there's another problem, which is that we are still not differentiating between chain environments and web app environments. mainnet vs testnet != production vs staging. But we need to change more stuff to fix that.